mlflow/server/js/src/experiment-tracking/components/experiment-page/hooks/useExperimentListQuery.ts

```diff
@@ -26,17 +27,45 @@ export const useInvalidateExperimentList = () => {
   };
 };
 
+function tagFilterToSql(tagFilter: TagFilter) {
+  switch (tagFilter.operator) {
+    case 'IS':
+      return `tags.\`${tagFilter.key}\` = '${tagFilter.value}'`;
+    case 'IS NOT':
+      return `tags.\`${tagFilter.key}\` != '${tagFilter.value}'`;
+    case 'CONTAINS':
+      return `tags.\`${tagFilter.key}\` ILIKE '${tagFilter.value}'`;
```

Comment:
i think we might need the `%` for ILIKE to behave correctly

```suggestion
      return `tags.\`${tagFilter.key}\` ILIKE '%${tagFilter.value}'%`;
```